pkg/gui/editors.go

```diff
@@ -16,11 +16,11 @@ func (gui *Gui) handleEditorKeypress(textArea *gocui.TextArea, key gocui.Key, ch
 		textArea.MoveCursorDown()
 	case key == gocui.KeyArrowUp:
 		textArea.MoveCursorUp()
-	case key == gocui.KeyArrowLeft && (mod&gocui.ModAlt) != 0:
+	case key == gocui.KeyArrowLeft && (mod&gocui.ModAlt) != 0 || ch == 'b' && (mod&gocui.ModAlt) != 0:
```

Comment:
To avoid repeating the ModAlt condition, I'd write this as
```suggestion
	case (key == gocui.KeyArrowLeft || ch == 'b') && (mod&gocui.ModAlt) != 0:
```